<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleIOWidget</class>
 <widget class="QWidget" name="SingleIOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>28</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1,1,1,3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QRadioButton" name="btn_status">
     <property name="text">
      <string>RadioButton</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="MaskWheelComboBox" name="comboBox_funcList">
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBox_reverse">
     <property name="text">
      <string>Reverse</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_pin">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_note">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaskWheelComboBox</class>
   <extends>QComboBox</extends>
   <header>maskwheelcombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
